
> api@1.0.0 dev
> node --watch src/index.js

db:  Database {}
DBConnectorSQLite3: Connected to ./data/db.sqlite3
node:events:497
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE: address already in use :::3000
    at Server.setupListenHandle [as _listen2] (node:net:1908:16)
    at listenInCluster (node:net:1965:12)
    at Server.listen (node:net:2067:7)
    at Function.listen (/home/fc1/git-repos/praxisprojekt-EventAufgabenPlaner/app/api/node_modules/express/lib/application.js:635:24)
    at file:///home/fc1/git-repos/praxisprojekt-EventAufgabenPlaner/app/api/src/index.js:50:5
Emitted 'error' event on Server instance at:
    at emitErrorNT (node:net:1944:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:90:21) {
  code: 'EADDRINUSE',
  errno: -98,
  syscall: 'listen',
  address: '::',
  port: 3000
}

Node.js v22.9.0
Failed running 'src/index.js'

> api@1.0.0 dev
> node --watch src/index.js

db:  Database {}
DBConnectorSQLite3: Connected to ./data/db.sqlite3
Server is running on port 3000
DBService.update users db2bf548-44e7-4100-b0cd-b99fdecbe2ac {
  email: 'wlamperd3c@goo.ne.jp',
  isActive: 0,
  isAdmin: 0,
  passwordHash: '9f6ebe1c6d351d4e581447fcee1fe06a8d882e2fbf0fdf5eea0b043875bc2be2'
}
SQL: UPDATE users SET email = ?, isActive = ?, isAdmin = ?, passwordHash = ? WHERE id = 'db2bf548-44e7-4100-b0cd-b99fdecbe2ac'
Werte: [
  'wlamperd3c@goo.ne.jp',
  0,
  0,
  '9f6ebe1c6d351d4e581447fcee1fe06a8d882e2fbf0fdf5eea0b043875bc2be2'
]

> api@1.0.0 dev
> node --watch src/index.js

db:  Database {}
DBConnectorSQLite3: Connected to ./data/db.sqlite3
Server is running on port 3000
